requirements/edx/custom.txt

```diff
@@ -1,6 +1,7 @@
 # Custom requirements to be customized by individual OpenEdX instances
 
 # When updating a hash of an XBlock that uses xblock-utils, please update its version hash in github.txt.
+-e git+https://github.com/edx/xblock-utils.git@08e5a5a9fc8ab46b627435427fd7e04c20809009#egg=xblock-utils
```

Comment:
@antoviaque the proper fix should be done in upstream ansible config, Somewhere in [deploy.yml](https://github.com/edx/configuration/blob/368b717b4ac8dae96a9db6af4ab3aea4b4618c91/playbooks/roles/edxapp/tasks/deploy.yml), and I don't think that's something we can do really quick (like, for this release)